ci-operator/step-registry/mco/confd2/create-custom-mcp/mco-confd2-create-custom-mcp-ref.metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"path": "mco/confd2/create-custom-mcp/mco-confd2-create-custom-mcp-ref.yaml",
```

Comment:
How about rename the path to `mco/conf/day2` for easy-reading.

Reply:
from the default naming pattern. like `ipi-conf`, `conf` is used for day1 steps. we want to following same pattern for day2 as well.

Reply:
`mco/conf/day1`
`mco/conf/day2`
something like this, put day1/2 dirs under conf dir

Reply:
cc: @sergiordlr WDYT

Reply:
> something like this, put day1/2 dirs under conf dir

That is okay for me, too.

Reply:
Done. The steps were moved to conf/day1 and conf/day2 directories.